### PR TITLE
 Adding doc template to function refitElec

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
Inputs and Outputs are still to be determined

### CODE REVIEW

File: src/Functions/refitElec.m [hunk 1]
```diff
@@ -1,27 +1,35 @@
...
+%
+%    [refitReZeroedElecMod, filteredIntensity, skelScaleMm] 
+%    = refitElec(initialPoly, pointCloudWorld, voxelValues, varargin)
 %
-% Params: initialPoly     - Dx3 double, R3 ploynomial coeffecients, 
-%         pointCloudWorld - Nx3 double, point cloud of sourounding voxels in respective coordinate system
-%         voxelValues     - Nx1 double, voxels intensitiy values (1:1 to  pointCloudWorld)
-%         varargin        - options cell
+% INPUTS: 
+%    initialPoly:           Dx3 double, R3 ploynomial coeffecients
```
Comment (Daniel-Tojal) [inline src/Functions/refitElec.m:14]: @adhusch Could you please confirm the existing descriptions and provide descriptions for the missing ones?

Comment (adhusch) [reply src/Functions/refitElec.m:14]: confirmed.

filteredIntensity: 1D intensity profile
skelScaleMm: estimated mm per skeleton step

Comment (Daniel-Tojal) [reply src/Functions/refitElec.m:14]: Thank you very much!

**Comment by laurentheirendt** [comment]
Great @Daniel-Tojal 👍 ⚽️

**Pull request merged**